app/upgrades/v8/msg_filter_ante.go

```diff
@@ -0,0 +1,36 @@
+package v8
+
+import (
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	superfluidtypes "github.com/osmosis-labs/osmosis/v7/x/superfluid/types"
+)
+
+// MsgFilterDecorator defines an AnteHandler decorator for the v8 upgrade that
+// provide height-gated message filtering acceptance.
+type MsgFilterDecorator struct{}
+
```

Comment:
The NewMsgFilterDecorator() I describe in my comment above (https://github.com/osmosis-labs/osmosis/pull/1481#discussion_r873071605) would go here

Reply:
Created new issue [here](https://github.com/osmosis-labs/osmosis/issues/1487#issue-1236190344) to address this in a separate PR after v8 upgrade to avoid an addition round of code reviews. Resolving this discussion for now to remove clutter from this PR